frb/builds/build_frbs.py

```diff
@@ -21,10 +21,18 @@ def frb_121102():
                         z_frb=0.19273)
     # NE2001
     frb121102.set_DMISM()
+
+    # FRB properties
+    frb121102.Fluence = 0.1 * units.Jy * units.ms    # Lower limit from Chatterjee+17/Bhandari+20
```

Comment:
Fluence is lower case in the code.  Be sure to
look at `GenericFRB` class in `frb.py`